lib/web_ui/lib/src/engine/text/paragraph.dart

```diff
@@ -1113,6 +1128,13 @@ void _applyParagraphStyleToElement({
     if (style._fontFamily != previousStyle._fontFamily) {
       cssStyle.fontFamily = quoteFontFamily(style._fontFamily);
     }
+    if (style._shadows != previousStyle._shadows) {
```

Comment:
Comparing `List<ui.Shadow>` using `!=` won't work because `List` does not implement `operator==` properly. We probably need a `listsEqual` utility that compares lists element-wise.

Reply:
Unfortunately no ImmutableList there but TextStyle is not mutable itself, so keeping identity check.